src/test/storage.zig

```diff
@@ -33,12 +61,23 @@ pub const Storage = struct {
         /// Average number of ticks it may take to write data. Must be >= write_latency_min.
         write_latency_mean: u64,
 
-        /// Chance out of 100 that a read will return incorrect data, if the target memory is within
-        /// the faulty area of this replica.
-        read_fault_probability: u8,
-        /// Chance out of 100 that a read will return incorrect data, if the target memory is within
-        /// the faulty area of this replica.
-        write_fault_probability: u8,
+        /// Chance out of 100 that a read will corrupt a sector, if the target memory is within
+        /// a faulty area of this replica.
+        read_fault_probability: u8 = 0,
```

Comment:
Elsewhere we use floats for probabilities (eg cluster.HealthOptions). Why the u8 here?

Reply:
Because I didn't realize that when I wrote `HealthOptions`. :wink: We want to eventually migrate away from using any floats at all though — they aren't deterministic between different CPUs, and we want all of our simulators/fuzzers to be fully deterministic.

Reply:
Hmmm. Maybe I should avoid them in test/fuzz too.